### PR TITLE
docs: Changes due to updated MCDU Server feature

### PR DESCRIPTION
**Pull request by frankkopp**
Depends on: https://github.com/flybywiresim/a32nx/pull/6446

## Summary
Changes to MCDU Web Interface doc for https://github.com/flybywiresim/a32nx/pull/6446

### Location
Feature Guides

Discord username (if different from GitHub): Cdr_Maverick#6475

### CODE REVIEW

File: docs/fbw-a32nx/feature-guides/web-mcdu.md [hunk 1]
```diff
@@ -230,18 +230,12 @@ Sometimes the default ports 8080 or 8125 are already used by other services on y
...
 ```
-Error: listen EACCES: permission denied 0.0.0.0:8080
+or
 ```
-<!-- to be changed after update is done
 Error: Port 8080 is already in use
--->
+```
 
 ??? tip "How To Check If a Port is Already In Use?"
```
Comment (tyler58546) [inline docs/fbw-a32nx/feature-guides/web-mcdu.md]: I think the "How To Check If a Port is Already In Use?" section is redundant now.

Comment (frankkopp) [reply docs/fbw-a32nx/feature-guides/web-mcdu.md]: Not so sure. We had several reports of users who said the server continuously "waited for simulator". Only after changing the web socket port did it connect. Did you see those reports?

**Comment by tyler58546** [comment]
Can you update this with the new ports from flybywiresim/a32nx#6446?

**Comment by frankkopp** [comment]
> Can you update this with the new ports from [flybywiresim/a32nx#6446](https://github.com/flybywiresim/a32nx/pull/6446)?

Of course - will do.
But what do you think about not changing the browser IP address? That never caused trouble with users and changing it will break any bookmarks and may be confusing to users.

**Review by tyler58546 — APPROVED** [review]

**Pull request merged**